cli/src/ui/commands/Chat.tsx

```diff
@@ -78,7 +91,94 @@ const CliChat: FC<CliChatProps> = ({ sId: requestedSId }) => {
     setIsSelectingNewAgent(true);
   }, []);
 
-  const commands = createCommands({ triggerAgentSwitch });
+  const clearFiles = useCallback(() => {
+    setUploadedFiles([]);
+    setPendingFiles([]);
+    setIsUploadingFiles(false);
+  }, []);
+
+  const showAttachDialog = useCallback(async () => {
+    await clearTerminal();
+    setShowFileSelector(true);
+  }, []);
+
+  // Helper to create a conversation for file uploads if none exists
+  // Only useful for uploading files to the first message
+  const createConversationForFiles = useCallback(async () => {
```

Comment:
So if you upload a file in your first message your conversation will be named "File Upload" instead of the same name we use below ?

I wonder if we could centralize the place where we create the conversation ?

Reply:
changed the title of the conv create to use the name of the files
quickly looked into putting the two usages of dustClient.createConversation into a single function but that would basically just be a proxy to the dustClient, idk if that makes sense

i can remove createConversationForFiles and inline it if you prefer